research/src/pages/select/select.proposal.mdx

```diff
@@ -195,11 +195,19 @@ and is therefore considered to be invalid.
 - The currently selected `<option>`, or the first if one isn't selected, should be
   visible within the listbox. This may require moving the list to accomodate listbox positioning
   and available space.
-- The user can dismiss the listbox and remove the state of `open` from the `<select>` in any of the
+- The user can dismiss the listbox and remove the state of `open` from the `<select>` in either of the
   following manners:
-  - The user clicks outside of the listbox
-  - Presses the escape key
-  - Selects one, or more (if `multiple` is true), by clicking or hitting `enter` or `space` key
+  - Triggering any of the listbox's <a href="#light-dismiss">light dismiss</a> behaviors
+  - Selecting one, or more (if `multiple` is true), options by clicking or hitting the `enter` or `space` key
+
+### Light dismiss <a class="link" href="#light-dismiss" id="light-dismiss"></a>
+The listbox part has "light dismiss", behavior, defined as being dismissed (removing
+the `open` state of the `<select>`, in this case) by either of the following things:
+- The user presses the escape key.
+- Focus moves outside of the subtree of the element with <a href="#light-dismiss">light dismiss</a>
```

Comment:
Is saying "with a light dismiss" here somewhat circular logic? Perhaps we could either nix this phrase from the description (just "Focus moves outside of the subtree of the element") or be more specific about what kind of user input behaviors cause a light dismiss.

Reply:
The attempted meaning was that "the element" refers to "the element that has the property of being light-dismiss" aka the listbox, but this just makes things murkier so I'll just say "listbox".